renovate.json5

```diff
@@ -28,5 +28,10 @@
         enabled: false,
       },
     },
+    {
+      description: 'Ignore pinning for capacitor OCI repository tag',
```

Comment:
Please include the reason why, or add a link to the explanation, thanks!

Reply:
Good idea! I'll add this to the other description as well.

Reply:
Addressed in [d4c60b3](https://github.com/PHACDataHub/safe-inputs/pull/507/commits/d4c60b34764918a46cc9ca12c3b31b2522c63d1d). Hope that works!